command.go

```diff
@@ -848,6 +893,22 @@ func (c *Command) InitDefaultHelpFlag() {
 	}
 }
 
+// InitDefaultHelpFlag adds default version flag to c.
```

Comment:
Correct to `// InitDefaultVersionFlag ...`.

Reply:
Thanks -- done